src/Optical/Emitters/Directions.jl

```diff
@@ -0,0 +1,206 @@
+module Directions
+
+using ....OpticSim, ...Geometry
```

Comment:
I understand the desire to encapsulate definitions to avoid polluting the namespace but this may be taking it too far. Is it really necessary for these modules to be nested so deeply? This is not considered good Julia programming style.